### DISCUSSION TITLE
Unable to pass through "Please add NICs to the Monitor interface"

### DISCUSSION THREAD

**Discussion by DevEsteves** [Security-Onion-Solutions/securityonion#11270] [2.4] [answered]
### Version

2.4.0

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Eval

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

12

### Storage for /

200

### Storage for /nsm

100

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

I've currently done the [first part](https://youtu.be/BFdTHGcsmkk?si=WzhkIk1KiLXN0yLD) through the installation of security onion 2.4, but now I'm stuck at this one field from part 2, it's in the image below. It's doing nothing when I press enter, I've seen in the part 2 that a " * " should show up and then I would press ok. Also output for the command `sudo salt-call state.highstate` was command not found. 

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/54783087/65492490-e7e7-417a-ad08-f8aa0ba55dcd)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by jwmasekre** [comment]
The first page of the install script mentions that `spacebar` will select and deselect items. In order to select that NIC, press `space`, and then you should be able to hit enter to continue.

**Reply by dougburks** [reply, marked as answer]
From https://docs.securityonion.net/en/2.4/first-time-users.html:

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/1659467/33082019-cba9-4075-997e-d322316dcdbb)
